go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -17,7 +17,7 @@
       }],
       repo_contents_path: repo_contents_path,
       directory: "/",
-      tidy: tidy
+      options: { tidy: tidy, vendor: false }
```

Comment:
Nit: test scoping feels a little off given where the new functionality is.

Meh as `FileUpdater` spec covers the right bits and they're still ~maybe combineable.